src/ncomm/ipcKeyDbm.c

```diff
@@ -210,8 +210,8 @@ int ipcKeyGet(IPC_KEY_DBM_ID dbmId, char* idstr,IPC_KEY_DBM_DATA* entry)
     if ( strlen(idstr) > (size_t) IPCKEY_MAXSTR_LEN )
     {
        errLogRet(ErrLogOp,debugInfo,
-        "ipcKeyGet: Warning, search string '%s' (%d chars) is too long, maximum is %d\n",
-	 idstr,strlen(idstr),IPCKEY_MAXSTR_LEN);
+        "ipcKeyGet: Warning, search string '%s' is too long, maximum is %d\n",
```

Comment:
is C99 assumed yet?  could have also maybe used the new %zu for size_t param

Reply:
The gcc on CentOS 7.4 is version 4.8.5 and it uses gnu90, which they describe as C90 including some C99 features. It does include the %zu. However, I have never actually seen these particular errors messages displayed and decided to just eliminate the strlen value and avoid any potential portability issues.